### PR TITLE
dev/refactor: Refactor Tests & LoggingService

### PR DESCRIPTION
**Pull request by sgoudham**

### CODE REVIEW

**Review by AlanRobinson1337 — APPROVED** [review]

**Review by erinmaria98 — APPROVED** [review]
Looks good. Tests look great. Looks easier to read where lines of code have been reduced.

**Review by hzl616 — APPROVED** [review]
I like the swap of the success message for the dbUrl. This makes it more efficient to more scenarios

**Review by 2013884f — APPROVED** [review]
Good additions to the tests.
Use of the conditional operator in LoggingServiceImpl line 27 maks that much more readable.

File: src/test/java/org/somerville/swag/data/service/LoggingServiceImplTest.java [hunk 1]
```diff
@@ -25,13 +26,34 @@
...
     void init_mocks() {
         MockitoAnnotations.openMocks(this);
     }
 
+    @Test
+    void successfullyReturnSameInstance() {
+        LoggingService firstLoggingService = LoggingServiceImpl.getInstance();
+        LoggingService secondLoggingService = LoggingServiceImpl.getInstance();
+
+        assertSame(firstLoggingService, secondLoggingService);
```
Comment (RobertRanson) [inline src/test/java/org/somerville/swag/data/service/LoggingServiceImplTest.java:42]: Excellent testing of the singleton design pattern!

**Review by RobertRanson — COMMENTED** [review]
Really great tests here, good job! Much more readable after changing db url

**Pull request merged**